docs/contributing/pair-programming.md

```diff
@@ -30,8 +30,8 @@ We also expect the following from pair programming participants:
 
 ## How to sign up
 
-We allow booking up to 30 days in advance, and spots tend to fill up fast. If we’re all booked up today, check back tomorrow for the next batch of spots. The core team is spread across North America, India, and Europe, so most time zones will have availability during working hours.
+Fill in the [request a pairing session form][cal] to reserve your spot. The Gatsby team will be notified. You should get an email shortly with a calendar link where you can choose a spot that fits you and the organizer. The core team is spread across North America, India, and Europe, so most time zones will have availability during working hours.
```

Comment:
I think we need to pause sessions until we iron out how our new pairing setup will work. I think we can still take responses through the form but adjust expectations and let them know that the calls will be recorded from now on. We should also edit that information earlier in this file to reflect the new process.
```
Fill in the [request a pairing session form][cal] to reserve your spot. The Gatsby team will be notified. You should get an email shortly with next steps. 

**Note:** Pairing sessions are currently paused as we establish a new process that will include recording so we can create lasting resources for the community from these sessions. We'll reach out to set up a time that works for you once we have the details ironed out. The core team is spread across North America, India, and Europe, so most time zones will have availability during working hours.```